vhdbuilder/packer/install-dependencies.sh

```diff
@@ -236,11 +236,11 @@ for AZURE_CNI_NETWORKMONITOR_VERSION in ${AZURE_CNI_NETWORKMONITOR_VERSIONS}; do
 done
 
 AZURE_NPM_VERSIONS="
+v1.2.2_hotfix
 1.2.1
 1.1.8
 1.1.7
 1.1.5
-1.1.4
```

Comment:
still technically used by subscription 90878020-48fc-45be-90d0-e12eb86ad61c. CAn we remove that? Also should we yank .5 and .7 while we're here usually we cache about 3 versions.

Reply:
Internal search on the subid returns zero results, sure i can trim the version list.